common/src/main/java/org/keycloak/common/profile/PropertiesProfileConfigResolver.java

```diff
@@ -46,6 +46,6 @@ public static String getPropertyKey(Feature feature) {
     }
 
     public static String getPropertyKey(String feature) {
-        return "keycloak.profile.feature." + feature.replaceAll("-", "_");
+        return "keycloak.profile.feature." + feature.replaceAll("[-:]", "_");
```

Comment:
the parameter name for Profile.getFeatureConfig needs updated - it implies that only unversioned is being passed, but that's not the case.

Reply:
Good point, I'll fix that. Thanks.

Reply:
Done.